pyeudiw/tests/tools/test_sd_jwt.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from pyeudiw.jwk import JWK
+from pyeudiw.sd_jwt import (issue_sd_jwt, verify_sd_jwt, _adapt_keys)
+
+from sd_jwt.utils.yaml_specification import load_yaml_specification
+
+from sd_jwt.holder import SDJWTHolder
+
+
+def test_issue_sd_jwt():
+    issuer_jwk = JWK()
+    holder_jwk = JWK()
+    
+    user_claims_path = "./pyeudiw/tests/tools/specifications.yml"
+    
+    issue_sd_jwt(
+        user_claims_path, 
+        {"issuer": "http://test.com", "default_exp": 60, "specification_file": "./pyeudiw/tests/tools/specifications.yml", "no_randomness": True},
```

Comment:
please do not open a specification file on each call but load the specification in a constructor and use it runtime without any filesystem read access